### PR TITLE
CIS 6.1.4 to 6.1.9

### PR DESCRIPTION
**Pull request by satya-rajesh**
Added roles and verify for CIS 6.1.4 to 6.1.9

### CODE REVIEW

File: roles/cis/tasks/6/6.1.4.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: "6.1.4 Ensure permissions on /etc/group are configured (Scored)"
+  file:
+      dest: /etc/group
+      owner: root
+      group: root
+      mode: 0644
+  when: not verify|bool
```
Comment (ppadmavilasom) [inline roles/cis/tasks/6/6.1.4.yml]: this when is not needed as we switch at the include task level

File: roles/cis/tasks/6/6.1.7.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+- name: "6.1.7 Ensure permissions on /etc/shadow- are configured (Scored)"
+  file:
+      dest: /etc/shadow-
+      state: touch
+      owner: root
+      group: root
+      mode: 0600
+  when: not verify|bool
```
Comment (ppadmavilasom) [inline roles/cis/tasks/6/6.1.7.yml]: same here

File: roles/cis/tasks/6/6.1.8.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+- name: "6.1.8 Ensure permissions on /etc/group- are configured (Scored)"
+  file:
+      dest: /etc/group-
+      owner: root
+      group: root
+      mode: 0600
+  when: not verify|bool
```
Comment (ppadmavilasom) [inline roles/cis/tasks/6/6.1.8.yml]: here too

**Review by ppadmavilasom — COMMENTED** [review]
looks good. minor fixes. 
lets please wait to merge https://github.com/vmware/ansible-security-hardening/pull/6 and we can merge this in.

**Review by satya-rajesh — COMMENTED** [review]
Removed "-" for register variable as ansible is throwing an error while executing tests.

**Pull request merged**